wiki/BBCode/pl.md

```diff
@@ -0,0 +1,380 @@
+    - Część wyników będzie ukryta bazując na języku i lokalizacji.
+- Składnia:
+```
+[google]...[/google]
+```
+
+### Szczęsliwy traf
+
+Dodaje łącze do strony wyszukanej dzięki funkcji *Szczęśliwy traf* wyszukiwarki Google.
+
+
+- Dedykowany przycisk: Brak
+- Ważne:
+  - Warto zanotować, że wyniki wyszukiwania nie będą takie same dla wszystkich.
+    - Część wyników będzie ukryta bazując na języku i lokalizacji.
+- Składnia:
+```
+[lucky]...[/lucky]
+```
+
+### Lista
+
+Zmienia tekst w listę.
+
+W BBCode, listy składają się z dwóch części: znaczników - ukazujących zasięg listy - oraz punktów.
+
+- Dedykowany przycisk:
+  - Lista punktowana: ![Przycisk dla listy puntkowanej](img/list.png)
+  - Lista numerowana: ![Przycisk dla listy numerowanej](img/list-numbered.png)
+- Ważne:
+  - Wszystkie punkty muszą znajdować się wewnątrz znacznika Lista.
+  - Istnieje 6 argumentów dla atrybutu `TYPE`:
+    - _(pusty)_ - punkty
+    - `1` - liczby
+    - `a` - litery (małe litery)
+    - `A` - litery (duże litery)
+    - `i` - liczby rzymskie (małe litery)
+    - `I` - liczby rzymskie (duże litery)
+  - Listy umieszczone wewnątrz list mogą być problematyczne.
+- Składnia:
+  - `TYPE` to jeden z argumentów opisanych powyżej.
+  - Jeżeli `TYPE` nie jest określony, zostanie użyta lista punktowana.
+```
+[list=TYPE]
+[*]...
+[/list]
+```
+
+### Obrazek
+
+Dodaje obrazek z łącza.
+
+Obrazki mogą być załączane z każdej strony, dopóki znajdują się one w łączach.
+
+**Nie dodawaj obrazków używając lokalnej ścieżki dostępu!** Użycie `C:\Users\Name\Pictures\image.jpg` **nie zadziała**.
+
+Prosimy o dodawanie zdjęć z wiarygodnych stron internetowych takich jak [imgur](https://imgur.com/ "imgur") czy [puush](https://puush.me/ "puush"). Gdy zdjęcie zostanie przesłane, skopiuj bezpośredni odnośnik do obrazka i wstaw je między znacznik otwierający a zamykający. Miej na uwadze, że niektóre strony nie podają bezpośrednich odnośników do zdjęć. Strony, takie jak te podane powyżej, podają owe odnośniki, ponieważ są to strony oferujące darmowy hosting grafiki.
+
+- Dedykowany przycisk: ![Przycisk dla obrazka](img/image.png)
+- Ważne:
+  - Jeśli w poście występuje wiele zdjęć albo zdjęcia o dużym rozmiarze, zaleca się umieszczenie ich w [Bloku](#blok).
+- Składnia:
+  - `LINK` to bezpośredni odnośnik do zdjęcia.
+```
+[img]LINK[/img]
+```
+
+### Youtube
+
+Dodaje wbudowany odtwarzacz dla filmiku z YouTube'a.
+
+- Dedykowany przycisk: Brak
+- Ważne:
+  - Jeżeli w poście występuje wiele zdjęć bądź są one duże, zaleca się umieszczenie ich w [Bloku](#blok).
+- Składnia:
+  - `VIDEO_ID` to ID filmiku (11-literowe), **nie** cały link!
+    - ID filmiku jest umieszczone w linku po `?v=`.
+```
+[youtube]VIDEO_ID[/youtube]
+```
+
+### Nagłówek (v1)
+
+Dodaje duży różowy nagłówek.
+Zazwyczaj używany w celu zaznaczenia nowej sekcji w poście.
+
+- Dedykowany przycisk: ![Przycisk dla nagłówka](img/heading.png)
+- Ważne:
+  - Wymaga samodzielnego napisania.
```

Comment:
Treść nagłówka należy wpisać pomiędzy znacznikami.

Jak to nie brzmi ciekawie, to albo wymyśl coś własnego, albo wyjeb.